README.md

```diff
@@ -86,163 +85,175 @@ Some additionally important notes about how snap works:
 
 ## Getting Started
 
-### In Active Development
-The master branch is used for new feature development. Our goal is to keep it in a deployable state. If you're looking for the most recent binary that is versioned, please [see the latest release](https://github.com/intelsdi-x/snap/releases).
-
 ### System Requirements
-Snap deploys as a binary, which makes requirements quite simple. We've tested on a subset of Linux and OS X versions.
+
+Snap does not have external dependencies since it's compiled into a staticly linked binary. At this time, Snap only runs on MacOS and Linux. We generate RPM/Deb packages for Linux and pkgs for MacOS X. We also provide snapd/snapctl binaries for testing the latest build.
```

Comment:
Snap can run on Windows.  

~~At this time, Snap only runs on MacOS and Linux.~~
At this time, we build and package snap for Linux and MacOS X.